### PR TITLE
[IMP] sales: add expense category context to reinvoice

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/odoo/project.task/3934622?cids=3

- added necessary context in regards to expense categories to reinvoicing section of expense doc
- added new screenshots

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @brse-odoo --- this one is ready for a Peer Review, whenever you get a chance. Thanks! (It's basically just the added important admonition block to add relevant context to the doc, so it shouldn't be too cumbersome) 👍

File: content/applications/sales/sales/invoicing/expense.rst [hunk 1]
```diff
@@ -155,6 +155,34 @@ Upon clicking :guilabel:`Post Journal Entries`, that button disappears, and the
...
+   to view a list of all expense categories in the database.
+
+   Look in the :guilabel:`Re-Invoice Expenses` column to see which selections have been made for
+   each expense category.
+
+   .. image:: expense/expense-categories-page.png
+      :align: center
+      :alt: The Re-Invoice Expenses column on the Expense Categories page in the Odoo Expenses app.
+
+   To modify that selection for an expense category, select the desired category for the list to
```
Comment (brse-odoo) [inline content/applications/sales/sales/invoicing/expense.rst:176]:
```suggestion
   To modify the selection for an expense category, select the desired category for the list to
```

File: content/applications/sales/sales/invoicing/expense.rst [hunk 2]
```diff
@@ -155,6 +155,34 @@ Upon clicking :guilabel:`Post Journal Entries`, that button disappears, and the
...
+   reveal that specific expense from.
+
+   From here, under the :guilabel:`Invoicing` section, in the :guilabel:`Re-Invoice Expenses` field,
```
Comment (brse-odoo) [inline content/applications/sales/sales/invoicing/expense.rst:179]:
```suggestion
   Under the :guilabel:`Invoicing` section, in the :guilabel:`Re-Invoice Expenses` field,
```
Optional

**Review by brse-odoo — APPROVED** [review]
Hi @ksc-odoo I've finished reviewing this PR, great job! Nice add to the doc, approving now with a couple suggestions for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

**Comment by ksc-odoo** [comment]
Thanks for the helpful feedback, @brse-odoo -- I made all of your suggested edits, and believe this one is now ready for Final Review (@StraubCreative ) -- whenever you get a chance. Thanks! 👍

File: content/applications/sales/sales/invoicing/expense.rst [hunk 3]
```diff
@@ -155,6 +155,34 @@ Upon clicking :guilabel:`Post Journal Entries`, that button disappears, and the
 Distribution` column in the :guilabel:`Expense` tab is filled with the sales order that was
 initially configured to the expense in the :guilabel:`Customer to Reinvoice` field.
 
+.. important::
+   By default, the :guilabel:`Customer to Reinvoice` field is enabled for the :guilabel:`[TRANS &
+   ACC] Travel & Accomodation` expense category.
```
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/expense.rst:160]: Checking: is this the *only* expense category it's enabled for?

File: content/applications/sales/sales/invoicing/expense.rst [hunk 4]
```diff
@@ -155,6 +155,34 @@ Upon clicking :guilabel:`Post Journal Entries`, that button disappears, and the
...
+
+   Look in the :guilabel:`Re-Invoice Expenses` column to see which selections have been made for
+   each expense category.
+
+   .. image:: expense/expense-categories-page.png
+      :align: center
+      :alt: The Re-Invoice Expenses column on the Expense Categories page in the Odoo Expenses app.
+
+   To modify the selection for an expense category, select the desired category for the list to
+   reveal that specific expense from.
```
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/expense.rst:177]: Can make more concise, consider rewording.

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 
Looks good, thank you for getting to this so quickly and explaining it accurately. 

I just had a quick q below and a small note for your consideration.

Can move to TR when you're ready, thanks again!

File: content/applications/sales/sales/invoicing/expense.rst (hunk 3 above)
Comment (ksc-odoo) [reply content/applications/sales/sales/invoicing/expense.rst:160]: After looking into this, it appears that this field also appears for: [COMM] Communication, [FOOD] Meals, [MIL] Mileage, and it's not available for [EXP_GEN] Others or [GIFT] Gifts. Making that adjustment now. Thanks for bringing this to my attention.

**Comment by ksc-odoo** [comment]
Thanks for the always-helpful feedback, @StraubCreative -- I made those suggested adjustments, and believe this one is now ready for Tech Review @samueljlieber (whenever you get a chance) 👍

File: content/applications/sales/sales/invoicing/expense.rst [hunk 5]
```diff
@@ -155,6 +155,35 @@ Upon clicking :guilabel:`Post Journal Entries`, that button disappears, and the
 Distribution` column in the :guilabel:`Expense` tab is filled with the sales order that was
 initially configured to the expense in the :guilabel:`Customer to Reinvoice` field.
 
+.. important::
+   By default, the :guilabel:`Customer to Reinvoice` field is enabled for the :guilabel:`[TRANS &
+   ACC] Travel & Accomodation`, :guilabel:`[COMM] Communication`, :guilabel:`[FOOD] Meals`, and
```
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:160]: Typo :)
```suggestion
   ACC] Travel & Accommodation`, :guilabel:`[COMM] Communication`, :guilabel:`[FOOD] Meals`, and
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo, thanks for your work on this PR! Your changes look good, I am approving and delegating with one quick fix. Thanks!
..
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
Thanks, @samueljlieber -- I made that necessary modification, so this one is ready to go!

@robodoo r+

**Pull request closed without merging**